R/wwinference.R

```diff
@@ -189,7 +189,17 @@ wwinference <- function(ww_data,
     }
   }
 
-  return(out)
+  structure(out, class = "wwinference_fit")
+}
+
+#' @export
+print.wwinference_fit <- function(x, ...) {
+  cat("wwinference_fit object\n")
+  cat("Model fit object: ", x$fit, "\n")
```

Comment:
This doesn't work on list objects I don't think!

Reply:
Addressed in [4c7105b](https://github.com/CDCgov/ww-inference-model/pull/58/commits/4c7105bbda61a84c2401eb13a3bef6c2b49b1f96).